app/src/main/java/com/github/sdpteam15/polyevents/model/entity/Event.kt

```diff
@@ -162,19 +162,20 @@ data class Event(
      * @return string HH:MM
      */
     fun formattedStartTime(): String {
-        if (startTime == null) {
-            return ""
+        return if (startTime == null) {
+            ""
         } else {
             //return SimpleDateFormat("k:mm", Locale.getDefault()).format(startTime)
             val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("k:mm")
-            return startTime.format(formatter)
+            startTime.format(formatter)
         }
     }
```

Comment:
you can do better
with 
`import com.github.sdpteam15.polyevents.helper.HelperFunctions.apply`

```
fun formattedStartTime() =
        startTime.apply("") { it.format(DateTimeFormatter.ofPattern("k:mm")) }
```

Reply:
I dont think it is pertinent